### ISSUE TITLE
[Bug]: Missing class properties transform

### ISSUE THREAD

**Issue by dndeveloper-saurabh** [tsparticles/tsparticles#4799]
### Contact Details

saurabh@pustack.com

### What happened?

I am not able to build since the last update.
<img width="1577" alt="Screenshot 2022-11-01 at 12 35 56 PM" src="https://user-images.githubusercontent.com/97387703/199178713-f76f7cc0-fdfe-471f-83ec-ff62843caf2e.png">

### tsParticles Version

2.1.3

### Which library are you using?

React.js (react-particles, react-tsparticles, react-particles-js)

### tsParticles Configuration

```json
<Particles
          height={"calc(100vh - 142px)"}
          width="100%"
          params={{
            particles: {
              number: {
                value: 355,
                density: {
                  enable: true,
                  value_area: 789.1476416322727,
                },
              },
              color: {
                value: "ffffff",
              },
              shape: {
                type: "circle",
                stroke: {
                  width: 0,
                  color: "#000000",
                },
                polygon: {
                  nb_sides: 5,
                },
              },
              opacity: {
                value: 0.48927153781200905,
                random: false,
                anim: {
                  enable: true,
                  speed: 0.2,
                  opacity_min: 0,
                  sync: false,
                },
              },
              size: {
                value: 2,
                random: true,
                anim: {
                  enable: true,
                  speed: 2,
                  size_min: 0,
                  sync: false,
                },
              },
              line_linked: {
                enable: false,
                distance: 150,
                color: "#ffffff",
                opacity: 0.4,
                width: 1,
              },
              move: {
                enable: true,
                speed: 0.2,
                direction: "none",
                random: true,
                straight: false,
                out_mode: "out",
                bounce: false,
                attract: {
                  enable: false,
                  rotateX: 600,
                  rotateY: 1200,
                },
              },
            },
            interactivity: {
              detect_on: "canvas",
              events: {
                onhover: {
                  enable: true,
                  mode: "bubble",
                },
                onclick: {
                  enable: true,
                  mode: "push",
                },
                resize: true,
              },
              modes: {
                grab: {
                  distance: 400,
                  line_linked: {
                    opacity: 1,
                  },
                },
                bubble: {
                  distance: 83.91608391608392,
                  size: 1,
                  duration: 3,
                  opacity: 1,
                  speed: 3,
                },
                repulse: {
                  distance: 200,
                  duration: 0.4,
                },
                push: {
                  particles_nb: 4,
                },
                remove: {
                  particles_nb: 2,
                },
              },
            },
            retina_detect: true,
          }}
        />
```

### Relevant log output

```shell
Creating an optimized production build...
Failed to compile.

./node_modules/tsparticles-engine/esm/Core/Utils/Circle.js
SyntaxError: /Users/dndeveloper/Projects/Professional/PuStack/pustack-web-platform/node_modules/tsparticles-engine/esm/Core/Utils/Circle.js: Missing class properties transform.
  2 | import { getDistance } from "../../Utils/NumberUtils";
  3 | export class Circle extends Range {
> 4 |     radius;
    |     ^^^^^^^
  5 |     constructor(x, y, radius) {
  6 |         super(x, y);
  7 |         this.radius = radius;
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by matteobruni** [comment]
You typed version `2.1.3`, but the latest version is `2.4.0`. Do you get the error even with the latest version? How do I reproduce the error? `transform` is not even something coming from tsParticles classes

**Comment by dndeveloper-saurabh** [comment]
<img width="1338" alt="Screenshot 2022-11-01 at 3 02 28 PM" src="https://user-images.githubusercontent.com/97387703/199203807-2e8d3538-5deb-4fc1-904c-886fb691dbae.png">

We are using 2.1.3 version of react-tsparticles in our project since last year and it was working like a charm till yesterday. But since last evening, it is showing me this error. Can you help me with which kind of babel-loader I need?

I also tried the latest version but nothing worked.

As soon as I removed the react-tsparticles and the related code, Everything worked.

**Comment by matteobruni** [comment]
@dndeveloper-saurabh I need a way to reproduce this issue. My demo project is building like always.

**Comment by dndeveloper-saurabh** [comment]
@matteobruni This is the sample repository, the project code and related configs are removed but the package.json is the same.

As soon as I comment out the import statement of react-tsparticles in App.js, the app works.

https://github.com/dndeveloper-saurabh/tsparticles-bug-demo

**Comment by matteobruni** [comment]
@dndeveloper-saurabh I think this should help: https://stackoverflow.com/questions/35517245/error-missing-class-properties-transform

I'm going to fix this on my side in `2.5.0`. I try to release a fix later, since it's breaking builds. I'm going to change the `ESM` output to `ES2017`, I thought that `ESNext` would be fine, but it seems it wasn't a good idea.
